cluster/gce/util.sh

```diff
@@ -695,16 +677,14 @@ function kube-up {
   write-node-env
   create-node-instance-template
 
-  gcloud preview managed-instance-groups --zone "${ZONE}" \
+  gcloud beta compute instance-groups managed \
       create "${NODE_INSTANCE_PREFIX}-group" \
       --project "${PROJECT}" \
+      --zone "${ZONE}" \
       --base-instance-name "${NODE_INSTANCE_PREFIX}" \
       --size "${NUM_MINIONS}" \
       --template "${NODE_INSTANCE_PREFIX}-template" || true;
-  # TODO: this should be true when the above create managed-instance-group
-  # command returns, but currently it returns before the instances come up due
-  # to gcloud's deficiency.
-  wait-for-minions-to-run
+  gcloud beta compute instance-groups managed wait-until-stable "${NODE_INSTANCE_PREFIX}-group"
```

Comment:
does this need `|| true` at the end in case of errors? or some other error checking?